--- inetutils-2.3/am/libcurses.m4.orig	2022-06-18 20:34:05.000000000 +0200
+++ inetutils-2.3/am/libcurses.m4	2022-07-13 21:50:34.685765670 +0200
@@ -89,6 +89,7 @@ dnl contradictions.
 dnl
 AC_DEFUN([IU_LIB_TERMCAP], [
   AC_REQUIRE([IU_LIB_NCURSES])
+  AC_CHECK_LIB(tinfo, tgetent, [LIBTERMCAP=-ltinfo], [
   if test "$LIBNCURSES"; then
     LIBTERMCAP="$LIBNCURSES"
   else
@@ -149,6 +150,7 @@ AC_DEFUN([IU_LIB_TERMCAP], [
       AC_DEFINE([HAVE_TGETENT], 1, [Define to 1 if tgetent() exists.])
     fi
   fi
+  ])
   AC_SUBST(LIBTERMCAP)])dnl
 
 dnl IU_LIB_CURSES -- check for curses, and associated libraries
